### ISSUE TITLE
compiled rom either becomes a white screen or crashes

### ISSUE THREAD

**Issue by derryswagleton** [ILadis/geodashgba#1]
no idea why it does that, im using the last devkitpro version and already went through the getting started pages, i have never compiled any gba roms before so i might just be being dumb but i ran those make commands and it just gives me a blank rom
![imagen_2024-04-07_014806490](https://github.com/ILadis/geodashgba/assets/98282276/b3c5d284-2a9c-433d-954e-9cc061a393af)

**Comment by ILadis** [comment]
I've been developing this exclusively on Linux so it's very likely that there is something just not working when building on Windows.

Could you please revert to a clean checkout of the source code (which you can do by running `make purge` or by doing another `git clone`) and then rerun each make step. After doing so please copy & paste the complete output of your terminal here. Please also upload a TXT file containing the files and folders after all make commands have run (you can create such a file with `tree > files.txt` for example).

I'll gladly fix building on Windows if it doesn't involve a lot of changes. Because I can't test on Windows it's also expected to break again going forward.

Also if you just wanted to give the current version a try and did not intend to experiment with the source code or build any additional levels, I could also upload a prebuilt ROM from my development machine.

**Comment by derryswagleton** [comment]
yeah i just wanted to try it out on some real hardware i got, would be really cool if you sent a prebuilt rom but i might try building on linux if i someday try to add more levels and such

**Comment by ILadis** [comment]
As promised here is a prebuilt ROM: [geodash.zip](https://github.com/ILadis/geodashgba/files/14938428/geodash.zip)

Let me know if this works for you. Any feedback is welcomed.

**Comment by derryswagleton** [comment]
just tried out stereo madness on no$gba and i must say this is a really well done demake! i genuinly have nothing really bad to say about it, its just great, i'll try it on my gba with a flash cartridge as soon as i can, many thanks

**Comment by ILadis** [comment]
Thanks for the kind feedback.

Which flashcard are you using? Because if it happens to be an Everdrive you could also test loading levels directly from the SD card.

I've tested this on my hardware but it would be good to know if this also works for other people.

**Comment by derryswagleton** [comment]
im not entirely sure. got mine gifted from a close friend but the label worn off quite a bit

nonetheless it runs really well on original hardware, didnt get any freezes or anything, i'll try to fix the thing that makes the rom be blank myself since it mostly said something about nogba, which im guessing is no$gba but without the $, might've just been something i could easily change from a few lines of code
